Header header
string solv_status_long
string solv_status_lat
float64 solv_time_long
float64 solv_time_lat
float64[] xs
float64[] ys
float64[] vs
float64[] psis
float64[] xr
float64[] yr
float64[] vr
float64[] psir
float64[] df
float64[] acc
float64[] ss_fren
float64[] vs_fren
float64[] eys_fren
float64[] epsis_fren
float64[] curv
float64[] xr_recon
float64[] yr_recon
float64[] vr_recon
float64[] psir_recon
